frontend/src/components/profile/details/ProfileDetails.tsx

```diff
@@ -52,6 +56,81 @@ export const ProfileDetails = () => {
     <>
       <section className="bg-white">
         <div className="pt-6">
+          {(personnel?.chipsProfileMissing === true ||
+            Object.keys(personnel?.chipsIssues || {}).length > 0) && (
+            <Banner
+              type={BannerType.ERROR}
+              content={
+                <>
+                  <div className="flex justify-between items-center">
+                    <span className="font-bold text-errorRed">
+                      Government Profile Syncing Errors Detected
+                    </span>
+                    {personnel?.chipsLastPing && (
+                      <span className="text-sm text-gray-500">
+                        Profile data synced{' '}
+                        {differenceInDays(
+                          new Date(),
+                          offsetTimezoneDate(personnel.chipsLastPing),
+                        )}{' '}
+                        days ago
+                      </span>
+                    )}
+                  </div>
+                  <div className="mt-1">
+                    {personnel?.chipsProfileMissing ? (
+                      <p className="text-sm text-errorRed">
+                        This member&apos;s profile cannot be found in the government
+                        system.
+                      </p>
+                    ) : (
+                      <p className="text-sm text-errorRed">
+                        This member has some profile information flagged for review
+                        as they may contain errors. Please see the red tooltips for
+                        details and make necessary edits.&nbsp; The details with an
+                        exclamation icon are directly sourced from this member&apos;s
```

Comment:
The banner seems to contradict functionality since employeeID and paylistID are not editable.

Reply:
I do think those are ones we have to take CHIPS at face value for.